source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst

```diff
@@ -5,10 +5,79 @@
 Quick Start Guide
 =================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
+**COCO Sessions** sample app illustrates COCO Media SDK Android's ability to communicate in real time via text, audio, and video.
 
-   **Coming Soon**
+This tutorial will walk you through the process of setting up our example app and experience how we integrated the **COCO Media SDK**.
 
-..
-    <Suggestions from Ashish - Steps to Sync Sources, Compilation, and Running Successfully.>
+Step1: Clone the Sample App repo
+   - Visit `COCO Sessions Repo`_ on official `Elear Solutions`_ organization account, you can either clone using 
+
+     .. code-block:: shell
+  
+         git clone git@github.com:elear-solutions/cocomediasdk-android-video-conference-sample-app.git 
+
+    or `download source code <https://github.com/elear-solutions/cocomediasdk-android-video-conference-sample-app/archive/refs/heads/master.zip>`__.
+
+   - After unpacking the downloaded ZIP or cloning the repo we are good to
+     start with Android Studio.
+
+Step2: Verify Dependencies
+   -  Now that you have cloned/downloaded the code, import the project to Android Studio.
+
+   -  Verify that the **COCO Media SDK Android** and **Java** dependencies are included with latest versions:
+
+   .. code-block:: groovy
+      :caption: **app/build.gradle**
+   
+        implementation 'buzz.getcoco:cocomediasdk-android:<x.y.z>'
+        implementation 'buzz.getcoco:cocomediasdk-java:<a.b.c>-lite'
+
+Step3: Node Sample Token Server
+   - Run the node sample token server as suggested in pre-install checklist.
+   - Visit :ref:`guide for service apps <quick_start_guide_service_apps>` to learn more.
+   - Its time to launch the app, connect your Android device to your PC and click **Run** on Android Studio toolbar.
+
+Step4: Navigating Through Sample App
+   Sample app comprises of 5 screens, described below:
+      
+   .. figure:: /_static/coco_sessions_flow.png
+      :align: center
+
+   - LoginActivity
+      This screen demonstrates the Third-party login solution for service apps.
```

Comment:
This section assumes that the user has already read the sample app walkthrough which comes after this section. Change the activity descriptions such that they are from the end user's point of view e.g. 

"This screen allows the user to login with his COCO credentials."

"This screen allows the user to view the Media Sessions that he is a part of."

...and so on

Reply:
improved as per suggestion.